pkgs/fc/ceph/src/fc/ceph/tests/test_maintenance.py

```diff
@@ -111,8 +143,17 @@ def test_lockimage_created(locktoolcalls, rbdcalls, ceph_json_calls):
         )
 
 
-def test_tempfail_when_another_lockholder(locktoolcalls):
-    maintenance_task = fc.ceph.maintenance.nautilus.MaintenanceTasks()
+def test_tempfail_when_another_lockholder(locktoolcalls, maintenance_manager):
```

Comment:
and another option would be to introduce `maintenance_manager_nautilus_plus` oooor with a bit more foresight `maintenance_manager_legacy` and `maintenance_manager` (that only has nautilus+). The latter likely improves maintainability in the future.

Reply:
Done. WHile doing so, I discovered that there currently is no test against non-Nautilus MaintenanceManagers anyways, but I still prepared the `maintenance_manager_legacy` fixture for future use.